bot.js

```diff
@@ -95,12 +115,17 @@ function replyText(text, message) {
 }
 
 function handleMessage(message) {
-  if (bot_name===undefined) {
-    setTimeout(function(){
+  if (bot_name === undefined) {
+    setTimeout(function() {
       handleMessage(message);
     }, 3000);
     return;
   }
+  
+  if (message.chat.type === 'private' && message.sticker) {
+    replyText('Sticker file_id: ' + message.sticker.file_id, message);
+  }
```

Comment:
Finding the sticker ID in privatecChat to use it in the stickers array. There currently is no way to find the sticker ID in the client.

Yes, this surely is not a good coding style to put this there undocumented …